packages/flutter/lib/fix_data.yaml

```diff
@@ -0,0 +1,134 @@
+# Spec file for "dart fix", see <insert URL>.
+
+version: 1
+transforms:
+  # Changes from https://github.com/flutter/flutter/pull/44189.
+  - title: 'Rename to dependOnInheritedElement'
+    date: 2019-11-22
+    element:
+      uris: [ 'material.dart', 'cupertino.dart', 'widgets.dart' ]
+      inClass: 'BuildContext'
+      method: 'inheritFromElement'
+    changes:
+      - kind: 'rename'
+        newName: 'dependOnInheritedElement'
+  - title: 'Migrate to dependOnInheritedWidgetOfExactType'
```

Comment:
As a drive by comment, you may want liberal use of whitespace and `#` comments to help separate the individual refactors. This file will likely get pretty large and will be tough to navigate w/o some visual separation between fixes.

Reply:
Agreed.